vividus-plugin-json/src/main/resources/vividus-plugin/spring.xml

```diff
@@ -12,16 +12,17 @@
 
     <bean id="json-context" class="org.vividus.json.variable.JsonContextVariable" />
 
-    <bean id="jsonSteps" class="org.vividus.json.steps.JsonSteps">
-        <constructor-arg>
-            <bean class="org.vividus.json.softassert.JsonSoftAssert" parent="softAssert" />
-        </constructor-arg>
-    </bean>
```

Comment:
why is this change needed?

Reply:
To move definition of JsonSoftAssert bean from definition of jsonSteps bean